hphp/hack/src/errors/errors.ml

```diff
@@ -3177,10 +3177,15 @@ let not_found_suggestion orig similar kind =
     end
   | (`instance, pos, v) -> suggestion_message orig v pos
 
-let snot_found_suggestion orig similar =
+let snot_found_suggestion orig similar kind =
   match similar with
   | (`instance, pos, v) ->
-    suggestion_message ~modifier:"instance method " orig v pos
+    begin
+      match kind with 
+      | `static_method -> suggestion_message ~modifier:"instance method " orig v pos
+      | `class_constant -> suggestion_message ~modifier:"instance property " orig v pos
+      | `class_variable | `class_typeconst -> suggestion_message orig v pos
+    end
```

Comment:
Please run `ocamlformat` so everything stays nicely formatted. It'll give you this:

```suggestion
    begin
      match kind with
      | `static_method ->
        suggestion_message ~modifier:"instance method " orig v pos
      | `class_constant ->
        suggestion_message ~modifier:"instance property " orig v pos
      | `class_variable
      | `class_typeconst ->
	suggestion_message orig v pos
    end
```

Reply:
I ran `ocamlformat hphp/hack/src/errors/errors.ml` which gave me some output but the actual file is not changed at all. Could you please suggest what might have gone wrong here? I read into the documentation [here](https://github.com/ocaml-ppx/ocamlformat) and it seems like there is some styling option I should pass to the command as well? 
For now I am just going to manually format this part of the file